### PR TITLE
support code server asExternalUrl - to open the hub correctly

### PR DESCRIPTION
**Pull request by amirbilu**

### CODE REVIEW

**Comment by amirbilu** [comment]
image

File: src/globals/consts.ts [hunk 1]
```diff
@@ -157,3 +157,4 @@ export const TABNINE_APP_URL = "https://app.tabnine.com";
 export const URI_SCHEME_FILE = "file";
 export const BINARY_RESTART_EVENT = "binary-restart-event";
 export const MAX_SMALL_INTEGER_V8 = 2 ** 30; // Max number that can be stored in V8's smis (small integers)
+export const LOCAL_ADDRESSES = ["localhost", "127.0.0.1"];
```
Comment (assapir) [inline src/globals/consts.ts:160]: 0.0.0.0?🥲

**Review by assapir — APPROVED** [review]
Boba

File: src/globals/consts.ts (hunk 1 above)
Comment (amirbilu) [reply src/globals/consts.ts:160]: Tachles.. we format the address.. and we hard-coded to 127.0.0.1

**Pull request merged**